runtime/src/main/java/com/couchbase/quarkus/extension/runtime/jacksonhandling/JacksonSupport.java

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
A lot of these classes don't mention our shaded names at all.  Is there a reasonable path to importing them directly instead?
Bit concerned about the maintenance burden of keeping so many C&Ped classes up-to-date.  Netty and Jackson are common CVE vectors and change regularly, and we update these dependencies a lot.  Presumably we will now need to recheck all these substitutions haven't changed, each time.
However I appreciate that rewriting each class to import some packages directly from Quarkus could actually be more work and maintenance.
Let me know your thoughts.